config/complete/orderly-web.yml

```diff
@@ -127,6 +127,9 @@ web:
   ## Optional sass variables to override default styles. Should be a
   ## file path relative to the config file
   sass_variables: variables.scss
+  ## Optional custom logo image, any valid image format. Should be a
+  ## file path relative to the config file.
+  logo: my-logo.jpg
```

Comment:
could you make this a png as it contains jpeg artefacts and I'd expect most people would have a png input (but they might convert to jpeg if they see the example has one)